redis/connection.py

```diff
@@ -715,7 +715,7 @@ class UnixDomainSocketConnection(Connection):
     def __init__(self, path='', db=0, password=None,
                  socket_timeout=None, encoding='utf-8',
                  encoding_errors='strict', decode_responses=False,
-                 retry_on_timeout=False,
+                 retry_on_timeout=False, socket_connect_timeout=None,
```

Comment:
I don't know whether that would work. The code I use in production and that I know works is the one I have in the comment I linked to above.